pgwidget/pgwidget_core.py

```diff
@@ -1420,9 +1421,12 @@ def on_unclick(self, glc):
         
 
 class Entry(TextContainerRect):
-    def __init__(self, text, pos, size, color=c.white, is_draggable=True, relative_pos=[0, 0],is_password=False,selection_color=c.red):
-        self.is_password=is_password #must be initialized before text definition (in super())
+    def __init__(self, text, pos, size, color=c.white, is_draggable=True, relative_pos=[0, 0],type='text',selection_color=c.red, file_types=None):
```

Comment:
@Omastto1 Breaking change - be careful about these - but probably meaningful - could you add docstring with explanation/examples + change "type" parameter to Literal/Enum so we know what options are available.